storageservice/Ballerina.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 org = "ballerinax"
 name = "azure_storage_service"
-version = "0.1.2-SNAPSHOT"
+version = "0.1.3-SNAPSHOT"
```

Comment:
Please dont change the connector version in the master branch. Leave it as it is.